eventhubs/client/azure-eventhubs/src/main/java/com/azure/eventhubs/EventHubConsumerOptions.java

```diff
@@ -14,85 +14,74 @@
     private Long priority;
     private Retry retryPolicy;
-    private boolean keepUpdated;
     private Scheduler scheduler;
     private int prefetchCount;
 
     /**
-     * Creates a new instance with a position at the beginning of the partition stream, consumer group set to
-     * {@link #DEFAULT_CONSUMER_GROUP_NAME},and the default prefetch amount.
+     * Creates a new instance with a position at the beginning of the partition stream, and the default prefetch amount.
      */
-    public EventReceiverOptions() {
-        this.consumerGroup = DEFAULT_CONSUMER_GROUP_NAME;
+    public EventHubConsumerOptions() {
         this.prefetchCount = DEFAULT_PREFETCH_COUNT;
     }
 
     /**
-     * Sets an optional text-based identifier label to assign to an event receiver.
+     * Sets an optional text-based identifier label to assign to an event consumer.
      *
      * @param identifier The receiver name.
-     * @return The updated ReceiverOptions object.
-     */
-    public EventReceiverOptions identifier(String identifier) {
-        validateIdentifier(identifier);
-        this.identifier = identifier;
-        return this;
-    }
-
-    /**
-     * Sets the name of the consumer group.
-     *
-     * @param consumerGroup The name of the consumer group.
-     * @return The updated ReceiverOptions object.
-     * @throws IllegalArgumentException If {@code consumerGroup} is {@code null} or an empty string.
+     * @return The updated {@link EventHubConsumerOptions} object.
+     * @throws IllegalArgumentException if {@code identifier} is greater than {@link #MAXIMUM_IDENTIFIER_LENGTH}.
      */
-    public EventReceiverOptions consumerGroup(String consumerGroup) {
-        if (ImplUtils.isNullOrEmpty(consumerGroup)) {
-            throw new IllegalArgumentException("'consumerGroup' cannot be null or empty.");
+    public EventHubConsumerOptions identifier(String identifier) {
+        if (!ImplUtils.isNullOrEmpty(identifier) && identifier.length() > MAXIMUM_IDENTIFIER_LENGTH) {
+            throw new IllegalArgumentException(String.format(Locale.US,
+                "identifier length cannot exceed %s", MAXIMUM_IDENTIFIER_LENGTH));
         }
 
-        this.consumerGroup = consumerGroup;
+        this.identifier = identifier;
         return this;
     }
 
+
     /**
-     * Sets the exclusiveReceiverPriority value on this receiver. When populated, the priority indicates that a receiver is intended to be
-     * the only reader of events for the requested partition and an associated consumer group.
-     * To do so, this receiver will attempt to assert ownership over the partition; in the case where more than one
-     * exclusive receiver attempts to assert ownership for the same partition/consumer group pair, the one having a
-     * larger {@link EventReceiverOptions#exclusiveReceiverPriority()} value will "win".
+     * Sets the {@code ownerLevel} value on this consumer. When populated, the priority indicates that a consumer
```

Comment:
nit:  `the priority` should probably read `the level`   or `the level indicates the priority of...`